packages/playwright-core/src/server/chromium/crBrowser.ts

```diff
@@ -196,6 +196,12 @@ export class CRBrowser extends Browser {
       context.emit(CRBrowserContext.CREvents.ServiceWorker, serviceWorker);
       return;
     }
+
+    // Detach from any targets we are not interested in.
+    // Ideally, detaching should resume any target, but there is a bug in the backend.
+    session._sendMayFail('Runtime.runIfWaitingForDebugger').then(() => {
```

Comment:
I don't think the severity of this issue justifies the workaround. At the very least it should link to an upstream bug, but ideally, we should close it once chromium is fixed.